tests/bluetooth/tester/src/btp_bap_broadcast.c

```diff
@@ -347,8 +351,11 @@ uint8_t btp_bap_broadcast_source_setup(const void *cmd, uint16_t cmd_len,
 	base_ad.type = BT_DATA_SVC_DATA16;
 	base_ad.data_len = ad_buf.len;
 	base_ad.data = ad_buf.data;
-	err = tester_gap_create_adv_instance(param, BTP_GAP_ADDR_TYPE_IDENTITY, &base_ad, 1, NULL,
-					     0, &gap_settings);
+	sd.type = BT_DATA_NAME_COMPLETE;
+	sd.data_len = strlen(dev_name);
+	sd.data = (const uint8_t *) dev_name;
```

Comment:
```suggestion
	sd.data = (const uint8_t *)dev_name;
```

We generally do not have spaces between the casting type and the value